third_party/amd/include/TritonAMDGPUToLLVM/PatternTritonAMDGPUToLLVM.h

```diff
@@ -0,0 +1,17 @@
+#ifndef THIRD_PARTY_AMD_INCLUDE_TRITONAMDGPU_TO_LLVM_PATTERNS_AMDGPU_OP_TO_LLVM_H
+#define THIRD_PARTY_AMD_INCLUDE_TRITONAMDGPU_TO_LLVM_PATTERNS_AMDGPU_OP_TO_LLVM_H
+
+#include "mlir/Conversion/LLVMCommon/TypeConverter.h"
+#include "triton/Analysis/AxisInfo.h"
+
+using namespace mlir;
```

Comment:
Please don't `use namespace` in header files--header files are meant to be included in other places and it will trigger we `use namespace` everywhere.